db/institutions.yml

```diff
@@ -1401,3 +1401,35 @@ zhejiangu:
   updated: 2020-02-14 19:37:28.925621
   uuid: 4f88b3c9-4719-4a0e-836b-16af6ab6605f
   year: 2020
+utokyo:
+  aka:
+    - The University of Tokyo
+  name: University of Tokyo
+  city: Tokyo
+  country: Japan
+icmm:
+  aka:
+    - Materials Science Institute of Madrid
+    - Institute of Materials Science of Madrid
+  name: Instituto de Ciencia de Materiales de Madrid
+  city: Madrid
+  country: Spain
+kyotou:
+  name: Kyoto University
+  city: Kyoto
+  country: Japan
+ucla:
+  aka:
+    - University of California, Los Anglelos
+  name: University of California
```

Comment:
This should be "University of California, Los Angeles" for name (also see typo above)